### PR TITLE
Created Crop/Word Cloud Container 

### PR DESCRIPTION
**Pull request by Kyleswillard**
## Description
Created div container for crop cloud image, and styled it to get as close to UI specifications as feasible. This component is needed to surround the word (crop) cloud and draw attention to it, and serve as a decoration. This is more appealing from a UI and UX perspective. 

Add a summary that address the following:

- motivation/purpose for the feature
 Stylize the container surrounding the crop/word cloud image 
 - What were those changes
 I created a div inside of the crop cloud div, and then adjusted width and height to ensure that there was the padding needed to surround the crop cloud image. Then I used border radius at 111px to create the marshmellow looking shape. 
- What issues were fixed
By adding this it breaks up the sea of white background, and brings the users attention to the crop cloud. 
- Provide the link to the user story it is referencing in Trello board or other supporting document
  https://trello.com/c/szLwC13f
-Loom Video
https://www.loom.com/share/6b33e1f561194a37bc662b6fbefe5deb

## Commits Checklist

- [ X] commits have clear title names
- [ X] commit has descriptive message of what has changed
- [ X ] more small commits than less big commits (single commit for minimal changes)

## Code Checklist

- [X ] code is formatted appropriately
- [X ] code meets DRY standards
- [X ] no dead code (check for logs and print statements)
- [X ] no TODOS instead necessary comments
- [X ] code follows Story Squad standards for:
  - Language
  - Framework
  - Libraries

### PRs needs to be reviewed by at least 2 team members

- TPL is the 3rd reviewer
- Lots of comments and suggestions, please!

[Pull Request Rubric](https://www.notion.so/1fc04e4fedeb429ba873b7c68d281707?v=74054da7991341c0bf970f39410c43da)

### CODE REVIEW

**Review by achaselittlefield — DISMISSED** [review]
Good job on making these changes and updating the word cloud. The only comment I have is possibly using a more responsive units such as rem versus using pixel for the height and width.

**Comment by Kyleswillard** [comment]
> Good job on making these changes and updating the word cloud. The only comment I have is possibly using a more responsive units such as rem versus using pixel for the height and width.

I certainly explored that. My argument for that is that we are well within the 50-60% mark, and that there will end up being an app for this, I would also suggest long term likely disabling this portion of the rendering with a media query. 

Good points for sure though.

File: src/components/pages/WordCloud/wordcloud.css [hunk 1]
```diff
@@ -69,10 +69,23 @@
   justify-content: center;
   align-items: center;
 }
+.crop-cloud-container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 615px;
```
Comment (tonysorensen) [inline src/components/pages/WordCloud/wordcloud.css:76]: When it comes to dimensions like height and width, use responsive units like ems, rems or percentages. You'll save yourself headaches down the road when you try to adapt things to different screen dimensions for mobile, tablet etc. Responsive units do the work for and require fewer media queries.

**Review by tonysorensen — DISMISSED** [review]

**Comment by Kyleswillard** [comment]
Resolved merge conflict.

**Review by achaselittlefield — APPROVED** [review]
Good job on this pull request. The commits are detailed and the loom video clarified any questions I would have.

**Review by ashtilawat — APPROVED** [review]

**Pull request merged**